package.json

```diff
@@ -67,5 +67,10 @@
     "posttest": "eslint . --ext .ts --config .eslintrc",
     "prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme",
     "version": "oclif readme && git add README.md"
+  },
+  "csdxConfig": {
+    "shortCommandName": {
+      "cm:stacks:validate-regex": "CTVLD"
```

Comment:
@abhinav-from-contentstack Is this the valid command `cm:stacks:validate-regex` in the content-type plugin?

Reply:
@abhinav-from-contentstack This commands seems to be not available in this plugin

Reply:
@netrajpatel I've corrected these command.